modules/amazon-s3-configuring-acs.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-amazon-s3.adoc
+:_module-type: PROCEDURE
+[id="amazon-s3-configuring-acs_{context}"]
+= Configuring {product-title}
```

Comment:
This is a pretty generic module title for what it's doing here. Could you possibly make it more specific, as in one of the following? 

* "Configuring an Amazon S3 integration in Red Hat ..." 
* "Integrating Red Hat ... with Amazon S3" 

Actually, as I come back, I'm questioning what this is actually doing. Is it both integrating RHACS with S3, and configuring auto backups? Are auto backups required in order to do the on-demand backups?

So if it's actually setting up auto backups, should it be titled more like "Configuring automatic backups ... "?

And if you don't actually have to set up the auto backups (and could just do the manual on-demand ones), should it be broken up into:

First, setting up the S3 integration, then separate modules for either configuring auto backups or performing an on-demand backup?

Reply:
The only reason anyone would integrate ACS with S3 is for automatic backups. Therefore, I'll go with your suggested heading "Configuring Amazon S3 integration in ..."